std_msgs/Header header
geometry_msgs/Vector3 latlongheight
geometry_msgs/Vector3 latlongheight_error
geometry_msgs/Vector3 llh_origin
geometry_msgs/Vector3 enu_position
geometry_msgs/Vector3 velocity # <course-over-ground> <speed-over-ground> <up speed>
geometry_msgs/Vector3 velocity_error
GpsStatus status
